nav2_system_tests/src/behaviors/wait/test_wait_behavior_launch.py

```diff
@@ -16,21 +16,42 @@
 import os
 import sys
 
+from os import environ
+from os import pathsep
+
 from ament_index_python.packages import get_package_share_directory
 
 from launch import LaunchDescription
 from launch import LaunchService
-from launch.actions import ExecuteProcess, IncludeLaunchDescription, SetEnvironmentVariable
+from launch.actions import DeclareLaunchArgument, ExecuteProcess
+from launch.actions import IncludeLaunchDescription, SetEnvironmentVariable
 from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
 from launch_ros.actions import Node
 from launch_testing.legacy import LaunchTestService
 
 from nav2_common.launch import RewrittenYaml
 
+from scripts import GazeboRosPaths
+
 
 def generate_launch_description():
-    map_yaml_file = os.getenv('TEST_MAP')
-    world = os.getenv('TEST_WORLD')
```

Comment:
Shouldn't `TEST_MAP`, `TEST_WORLD` and `GAZEBO_MODEL_PATH` be removed from `nav2_system_tests/src/behaviors/wait/CMakeLists.txt` as well?

Reply:
Done.